src/lib/ops/policyOps.js

```diff
@@ -145,18 +145,35 @@ const policyOps = {
    * @param  {Function} cb
    */
   listByOrganization: function listByOrganization (params, cb) {
-    const { organizationId } = params
+    const { organizationId, limit, page } = params
 
```

Comment:
So the limit and page end up as both `LIMIT` and `OFFSET` in SQL.
Postgres will validate that these are positive set for us with clear error messages. Is it necesssary to add our own validation code on top of it ?